RecoVertex/BeamSpotProducer/test/alcareco.py

```diff
@@ -65,7 +65,8 @@
 #process.GlobalTag.globaltag = 'IDEAL_V9::All'  # take your favourite
 
 process.GlobalTag.connect = "sqlite_file:/afs/cern.ch/user/f/futyand/public/globaltag/10PB_21X.db"
-process.GlobalTag.globaltag = '10PB_V1::All'
+#process.GlobalTag.globaltag = '10PB_V1::All'
```

Comment:
Please remove these

Reply:
sorry @francescobrivio where do you see the loading the GT as a local .db file as useful comment? That shouldnt really be used, right?

Reply:
Right, as discussed in https://github.com/cms-sw/cmssw/pull/38788#discussion_r925697919 we can remove alltogether this `RecoVertex/BeamSpotProducer/test/alcareco.py` file.